src/lib/routes/admin-api/client-metrics.ts

```diff
@@ -58,8 +59,11 @@ class ClientMetricsController extends Controller {
                 openApiService.validPath({
                     operationId: 'getRawFeatureMetrics',
                     tags: ['Metrics'],
+                    summary:
+                        'Feature usage metrics for the last 48 hours, grouped by hour',
```

Comment:
This feels like a mix of a summary and a description, but it's too long for the former and too short for the latter. Can we split it out?

```suggestion
                    summary:
                        'Get feature usage metrics for the last 48 hours',
                    description:
                        'Returns usage metrics from the last 48 hours for the specified feature. The metrics are grouped by hour.',
```